README.md

```diff
@@ -81,15 +81,19 @@ Check out the [ESLint docs](http://eslint.org/docs/rules/) for all possible rule
 
 - `sourceType` can be set to `'module'`(default) or `'script'` if your code isn't using ECMAScript modules.
 - `allowImportExportEverywhere` (default `false`) can be set to `true` to allow import and export declarations to appear anywhere a statement is allowed if your build environment supports that. Otherwise import and export declarations can only appear at a program's top level.
+- `ecmaFeatures.globalReturn` (default `false`) allow return statements in the global scope when used with `sourceType: "script"`.
 
-**.eslintrc**
+**.eslintrc.json**
 
 ```json
 {
   "parser": "babel-eslint",
   "parserOptions": {
     "sourceType": "module",
     "allowImportExportEverywhere": false,
+    "ecmaFeatures": {
+      "globalReturn": true
```

Comment:
why is this set to `true` and not the default, `false`?

Reply:
Ah, good catch. Will fix!